x-pack/plugins/observability_solution/observability_logs_explorer/public/components/alerts_popover.tsx

```diff
@@ -233,6 +248,7 @@ export const AlertsPopover = () => {
             iconType="arrowDown"
             iconSide="right"
             isLoading={canCreateRuleState.loading}
+            data-test-subj="logsExplorerToggleAlertsPopoverButton"
```

Comment:
**nit**: this button already has a `data-test-subj` property :)

Reply:
haha, didn't see that, will fix :D